TR-025 Incremento

RickGutler opened the pull request:
Feito o controller de Feira.
Criado os DTOs de Feira.
Feito o cadastro de Feira

RickGutler committed:
TR-025 Incremento

Feito o controller de Feira.
Criado os DTOs de Feira.
Feito o cadastro de Feira

Prezotti reviewed (changes requested):
O valor total também não precisa ser passado pelo front end, ele será calculado no próprio backend. Ou seja, o cadastro de feira não precisa receber nada, só retornar o detalhamento

RickGutler committed:
TR-025 Incremento

Feito o controller de Feira.
Criado os DTOs de  detalhamento da Feira.
Feito o cadastro de Feira
